iocage.8

```diff
@@ -1015,6 +1046,30 @@ The Source listed with each property shows whether it is a local
 .Nm
 property or where more information can be located.
 .Bl -tag -width "pkglist=none"
+.It Pf assign_localhost= Op 0 | 1
+Boolean option to add interface
+.Ql lo0
+assigned to
+.Ql 127.0.0.1 .
+Configured automatically when
+.Ql vnet=on.
```

Comment:
We should reword this as vnet jails setup their own localhost as they are an independent stack. The current text implies vnet=on sets this property. Which isn’t the case